great_expectations/rule_based_profiler/parameter_builder/metric_single_batch_parameter_builder.py

```diff
@@ -23,6 +23,12 @@ class MetricSingleBatchParameterBuilder(MetricMultiBatchParameterBuilder):
     and metric_name as arguments.
     """
 
+    exclude_field_names: Set[
+        str
+    ] = MetricMultiBatchParameterBuilder.exclude_field_names | {
```

Comment:
Is `single_batch_mode` a field? Maybe I don't understand what excluded fields are here.

Reply:
@billdirks This is for serialization/printing -- rooted in our reliance on `instantiate_class_from_config`.  Happy to discuss.